src/__tests__/core.test.ts

```diff
@@ -72,3 +72,34 @@ it('Test multiple blocks selected by specific hashes', async () => {
   const result = await graphql(schema, query);
   expect(result).toEqual(expected);
 });
+
+it('Test multiple blocks selected by number range', async () => {
+  const query = `
+    {
+      blocksRange(numbers: [10, 12]) {
+        timestamp
+      }
+    }
+  `;
+
+  const expected = { data: { blocksRange: [ { timestamp: "1438270128" }, { timestamp: "1438270136" }, { timestamp: "1438270144" }] }  };
+
+  const result = await graphql(schema, query);
+  expect(result).toEqual(expected);
+});
+
+it('Test multiple blocks selected by hash range', async () => {
+  const query = `
+    {
+      blocksRange(hashes: ["0x685b762b9d37807ab5c534936530afdb3794b79937f3e61e0b832d0e13e6eabf", "0x2e3d27de5a29082765794cd721c70fac641e546d683ccdc5d178e0bc2aca040e"]) {
+        number
+      }
+    }
+  `;
+
+  const expected = {
+    data: { blocksRange: [{ number: 20222 },{ number: 20223 },{ number: 20224 }] } };
+
+  const result = await graphql(schema, query);
+  expect(result).toEqual(expected);
+});
```

Comment:
I'm missing negative tests here. For example:

* Provide both number and hash ranges => error.
* Provide more than two numbers in the range => error.
* Provide more than two hashes in the range => error.
* Provide no numbers or hashes => error.
* Provide 1 number only => error.
* Provide 1 hash only => error.
* Provide a negative number in the range => error.
* Provide a start number larger than the end number => error.
* Provide a start hash whose block number is higher than the end number => error.
* Provide a hash that does not exist => error.
* Provide the same block number twice => it should just return one block.
* Provide the same hash twice => it should just return one block.